Gopkg.toml

```diff
@@ -151,3 +151,15 @@ required = [
 [[override]]
   name = "k8s.io/apiserver"
   version = "kubernetes-1.12.5"
+
+[[constraint]]
+  branch = "release-1.12"
+  name = "github.com/kubernetes-sigs/cri-o"
+
+[[constraint]]
+  branch = "master"
```

Comment:
can we have tags or something to avoid `dep` polluting everything when re-syncing?

Reply:
Changed to a revision